pom.xml

```diff
@@ -1113,6 +1113,27 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <!-- This is under Confluence Community License and it should not be used in compile scope -->
```

Comment:
Is there is any plugin that could fail if such libraries are added ?

Reply:
I am not aware of this. I tried to find such, but with no lock. I will define test scope in master pom.xml, so it will force to explicit overwrite and so there is high chance that someone would read the comment.